src/domain/follows_differ.rs

```diff
@@ -357,13 +352,19 @@ fn log_line(
 
     // Investigate states in which there are no followees but there are unfollowed followees
     if followed_counter == 0 && unfollowed_counter > 0 && unchanged == 0 {
+        metrics::sudden_follow_drops().increment(1);
```

Comment:
do these metrics calls automatically include metadata like the ID of the event that triggered it and/or the author pubkey? I think those would be really useful for post-mortems.

Reply:
That wouldn't be a good idea because of cardinality issues. In fact, we should get back to the event service and remove those metrics that include free form labels. But we do log each of these instances. So when we receive alerts, we can always track what happened through the logs.

Reply:
Oh I see. I finally looked up the definition of cardinality and it isn't what I thought. TIL 👍